float32 object_pos_x
float32 object_pos_y
float32 object_pos_z
float32 object_ori_w
float32 object_ori_x
float32 object_ori_y
float32 object_ori_z

float32 robot_pos_x
float32 robot_pos_y
float32 robot_pos_z
float32 robot_ori_w
float32 robot_ori_x
float32 robot_ori_y
float32 robot_ori_z
